src/clockwork/amqp.clj

```diff
@@ -30,4 +41,19 @@
     (log/info (format "[amqp/connect] [%s]" (config/amqp-uri)))
     (declare-exchange channel exchange-cfg)
     (declare-queue channel exchange-cfg queue-cfg (keys handlers))
-    (lc/blocking-subscribe channel (:name queue-cfg) (partial message-router handlers))))
+    (lc/blocking-subscribe channel (:name queue-cfg) (partial message-router handlers))
+    channel))
```

Comment:
This is possibly a relatively minor concern, but the [rabbitmq java client docs](http://www.rabbitmq.com/javadoc/com/rabbitmq/client/Channel.html) recommend a channel per thread, where this would reuse the channel. Some [best practices guides (e.g.)](https://dzone.com/articles/rabbitmq-amqp-channel-best) also recommend against using the same channel for publishing and consuming, which this would do. It would probably be best if this function returned a second channel made with an additional `lch/open` against the Connection returned by `rmq/connect`, or returned the connection directly and expected consumers to open new channels as needed.